robinhood_crypto_api/robinhood_crypto_api.py

```diff
@@ -244,3 +244,23 @@ def order_cancel(self, order_id):
 '9303.0050', 'volume': '0.0000', 'session': 'reg', 'interpolated': False}, {'begins_at': '2018-05-07T23:15:00Z', 'open_price': '9314.5600', 'close_price': '9308.1500', 'high_price': '9317.3100', 'low_price': '9304.2400', 'volume': '0.0000', 'session': 'reg', 'interpolated': False}, {'begins_at': '2018-05-07T23:20:00Z', 'open_price': '9308.1500', 'close_price': '9298.0000', 'high_price': '9311.0250', 'low_price': '9285.2200', 'volume': '0.0000', 'session': 'reg', 'interpolated': False}, {'begins_at': '2018-05-07T23:25:00Z', 'open_price': '9298.0000', 'close_price': '9326.0650', 'high_price': '9330.7700', 'low_price': '9296.3450', 'volume': '0.0000', 'session': 'reg', 'interpolated': False}, {'begins_at': '2018-05-07T23:30:00Z', 'open_price': '9326.0650', 'close_price': '9354.8250', 'high_price': '9360.4700', 'low_price': '9323.5300', 'volume': '0.0000', 'session': 'reg', 'interpolated': False}, {'begins_at': '2018-05-07T23:35:00Z', 'open_price': '9354.8250', 'close_price': '9347.6500', 'high_price': '9359.1450', 'low_price': '9327.0100', 'volume': '0.0000', 'session': 'reg', 'interpolated': False}, {'begins_at': '2018-05-07T23:40:00Z', 'open_price': '9347.6500', 'close_price': '9350.3350', 'high_price': '9357.1650', 'low_price': '9341.3800', 'volume': '0.0000', 'session': 'reg', 'interpolated': False}, {'begins_at': '2018-05-07T23:45:00Z', 'open_price': '9350.3350', 'close_price': '9339.4050', 'high_price': '9353.7450', 'low_price': '9336.2350', 'volume': '0.0000', 'session': 'reg', 'interpolated': False}, {'begins_at': '2018-05-07T23:50:00Z', 'open_price': '9339.4050', 'close_price': '9351.3950', 'high_price': '9357.2400', 'low_price': '9336.3850', 'volume': '0.0000', 'session': 'reg', 'interpolated': False}, {'begins_at': '2018-05-07T23:55:00Z', 'open_price': '9350.2450', 'close_price': '9361.4800', 'high_price': '9376.3850', 'low_price': '9350.2450', 'volume': '0.0000', 'session': 'reg', 'interpolated': False}, {'begins_at': '2018-05-08T00:00:00Z', 'open_price': '9361.4800', 'close_price': '9376.0350', 'high_price': '9382.5100', 'low_price': '9351.3450', 'volume': '0.0000', 'session': 'reg', 'interpolated': False}, {'begins_at': '2018-05-08T00:05:00Z', 'open_price': '9372.4100', 'close_price': '9374.3750', 'high_price': '9392.6650', 'low_price': '9372.1750', 'volume': '0.0000', 'session': 'reg', 'interpolated': False}, {'begins_at': '2018-05-08T00:10:00Z', 'open_price': '9374.3750', 'close_price': '9374.0050', 'high_price': '9377.6900', 'low_price': '9367.0550', 'volume': '0.0000', 'session': 'reg', 'interpolated': False}, {'begins_at': '2018-05-08T00:15:00Z', 'open_price': '9374.0050', 'close_price': '9376.0500', 'high_price': '9380.6300', 'low_price': '9364.7750', 'volume': '0.0000', 'session': 'reg', 'interpolated': False}], 'bounds': '24_7', 'interval': '5minute', 'span': 'day', 'symbol': 'BTCUSD', 'id': '3d961844-d360-45fc-989b-f6fca761d511', 'open_price': None, 'open_time': None, 'previous_close_price': None, 'previous_close_time': None}
```

Comment:
Response too long, trim it?
{  
   'data_points':[  
      {'begins_at':'2018-05-07T00:20:00Z', 'open_price':'9636.2650', 'close_price':'9598.4300', 'high_price':'9638.0600', 'low_price':'9594.3700', 'volume':'0.0000', 'session':'reg', 'interpolated':False },
      {'begins_at':'2018-05-07T00:25:00Z', 'open_price':'9594.3300', 'close_price':'9596.5100', 'high_price':'9604.2350', 'low_price':'9585.2950', 'volume':'0.0000', 'session':'reg', 'interpolated':False },
      ...
      {'begins_at':'2018-05-08T00:15:00Z', 'open_price':'9374.0050', 'close_price':'9376.0500', 'high_price':'9380.6300', 'low_price':'9364.7750', 'volume':'0.0000', 'session':'reg', 'interpolated':False }
   ],
   'bounds':'24_7',
   'interval':'5minute',
   'span':'day',
   'symbol':'BTCUSD',
   'id':'3d961844-d360-45fc-989b-f6fca761d511',
   'open_price':None,
   'open_time':None,
   'previous_close_price':None,
   'previous_close_time':None
}